DSA-session-3/src/question2/ArrayPriorityQueue.java

```diff
@@ -0,0 +1,128 @@
+package question2;
+
+/**
+ * This class is for Priority Queue Operations.
+ * @author Aman Gautam
+ * Dated  18/08/2019
+ */
+public class  ArrayPriorityQueue  implements PriorityQueue{
```

Comment:
Since it is defining the implementation of PriorityQueue methods, so the name could be `PriorityQueueImpl`